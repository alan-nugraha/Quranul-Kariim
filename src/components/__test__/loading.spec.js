import React from 'react';
import {render} from '@testing-library/react-native';
import Loading from '../loading';

describe('Loading', () => {
  beforeEach(() => {});

  afterEach(() => {
    jest.clearAllMocks();
  });

  afterAll(() => {
    jest.restoreAllMocks();
  });

  test('should render Loading correctly', () => {
    const {toJSON} = render(<Loading />);

    expect(toJSON()).toMatchSnapshot();
  });
});
