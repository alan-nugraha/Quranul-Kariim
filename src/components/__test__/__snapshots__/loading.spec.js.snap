// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render Loading correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(255,255,255,0.5)",
      "bottom": 0,
      "flex": 1,
      "height": "100%",
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "width": "100%",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#07A9F0"
    hidesWhenStopped={true}
    size={30}
  />
</View>
`;
